wrappers/wrapper.rb

```diff
@@ -306,7 +307,7 @@ def assert_clustering_compatible_with_periodic_heuristic(vrp)
     end
 
     def assert_lat_lon_for_partition(vrp)
-      vrp.configuration.preprocessing.partition_technique.nil? || vrp.points.all?{ |pt| pt.location && pt.location.lat && pt.location.lon }
+      vrp.configuration.preprocessing.partitions&.empty? || vrp.points.all?{ |pt| pt.location && pt.location.lat && pt.location.lon }
```

Comment:
I think this doesn't work if `preprocessing` or `partitions` is `nil` themselves. 

I do not know if "they being nil" is possible, if not, we don't need the `&` (but this is the case for other asserts too); otherwise (if it is possible), then we need something like the following:
```suggestion
      vrp.configuration.preprocessing&.partitions.to_a.empty? || vrp.points.all?{ |pt| pt.location && pt.location.lat && pt.location.lon }
```

I think if we can make sure that the model objects are correctly initialized with sensible default values (like empty `:partitions` etc) that would be nice but (it is out of scope of this PR) in the mean the above should do the trick I think.

Reply:
Nice catch ! Preprocessing could definitely be nil